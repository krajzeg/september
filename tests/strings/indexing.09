str := "Orange"

print("This should say 'O e':", str[0], str[5])
print("This should say 'g r':", str[-2], str[-5])
