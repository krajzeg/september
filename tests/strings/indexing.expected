This should say 'O e': O e
This should say 'g r': g r
